app/controllers/posts_controller.rb

```diff
@@ -375,6 +379,13 @@ def require_permission
     end
   end
 
+  def require_import_permission
```

Comment:
What if we just made this a before_action and returned unless view=import || button_import.present? Might be cleaner?

Reply:
Done! Looks to solve the other comment too.